pkg/ddevapp/hostname_mgt.go

```diff
@@ -110,13 +110,16 @@ func (app *DdevApp) AddHostsEntriesIfNeeded() error {
 			util.Warning("Unable to open hosts file: %v", err)
 			continue
 		}
-		util.Warning("The hostname %s is not currently resolvable, trying to add it to the hosts file", name)
-
-		out, err := escalateToAddHostEntry(name, dockerIP)
-		if err != nil {
-			return err
+		if strings.Contains(name, `*`) {
+			util.Warning("DDEV cannot add unresolvable wildcard hostnames like `%s` to your hosts file", name)
```

Comment:
Could this also point to docs or mention that there are probably ways to resolve this? The message is completely true, but still makes me think I misconfigured the project rather than having a general networking issue.

Reply:
Added link to new doc.